File: app/src/main/java/com/djavid/bitcoinrate/model/heroku/ResponseTickers.java
package com.djavid.bitcoinrate.model.heroku;

import java.util.List;


public class ResponseTickers {

    public String error;
    public List<Ticker> tickers;


    public ResponseTickers(String error) {
        this.error = error;
    }

    public ResponseTickers(String error, List<Ticker> tickers) {
        this.error = error;
        this.tickers = tickers;
    }

    public ResponseTickers(List<Ticker> tickers) {
        this.tickers = tickers;
        error = "";
    }

}

File: app/src/main/kotlin/com/djavid/bitcoinrate/model/heroku/ResponseTickers.kt
package com.djavid.bitcoinrate.model.heroku


class ResponseTickers {

    var error: String
    var tickers: List<Ticker>


    constructor(error: String) {
        this.error = error
    }

    constructor(error: String, tickers: List<Ticker>) {
        this.error = error
        this.tickers = tickers
    }

    constructor(tickers: List<Ticker>) {
        this.tickers = tickers
        error = ""
    }

}
